usecases/data_model_usecase.go

```diff
@@ -162,42 +173,191 @@ func (usecase *DataModelUseCase) CreateDataModelField(ctx context.Context, field
 				return usecase.organizationSchemaRepository.CreateField(ctx, orgTx, table.Name, field)
 			},
 		)
-	})
-	return fieldId, err
+	}); err != nil {
+		return "", err
+	}
+
+	if field.IsUnique {
+		if err := usecase.clientDbIndexEditor.CreateUniqueIndexAsync(
+			ctx,
+			getFieldUniqueIndex(models.TableName(tableName), field.Name)); err != nil {
+			return "", err
+		}
```

Comment:
Same here. Personally, this "shorthand" is meaningfull when it's a short one liner.
IMO, as soon as it's spread accross multiple lines, it's difficult to read.

Reply:
👍  true, I pushed it a bit far here